src/Nancy/ResponseExtensions.cs

```diff
@@ -182,7 +182,8 @@ public static Response WithStatusCode(this Response response, int statusCode)
         private static Tuple<string, string> GetTuple(object header)

```

Comment:
Unrelated to the PR, but brownie points if you get rid of the weird indentations in this method :heart_eyes: